CHANGELOG.md

```diff
@@ -1,5 +1,59 @@
 ## [Unreleased][unreleased]
 
+## [0.11.0rc2] - 2017/07/26
+
+This new RC addresses a few performance regressions spotted in 0.11.0rc1, and
+bumps the compatible versions of OpenResty up to the latest `1.11.2.4`.
+
+It also includes support for regexps URIs and a new `kong prepare` command.
+
+We are **actively** looking for testing and feedback regarding this release
+candidate!
+
+### Changed
+
+- :warning: Migrations are **not** executed automatically by `kong start`
+  anymore. Migrations is now a **manual** process, via the `kong migrations`
+  command. This is to enforce the "single node running migrations" rule.
+  [#2421](https://github.com/Mashape/kong/pull/2421)
+
+### Added
+
+- :fireworks: Support for regexps URIs! You can now define regexps in your APIs
+  `uris` property. Those regexps can have capturing groups which can be
+  extracted by Kong during a request, and accessed later in the plugins (useful
+  for URI rewriting).
+  [#2681](https://github.com/Mashape/kong/pull/2681)
+- Bump OpenResty compatibility to `1.11.2.4`. OpenResty's LuaJIT can now be built
+  with Lua 5.2 compatibility.
+  [#2489](https://github.com/Mashape/kong/pull/2489)
+- New `kong prepare` command to prepare the Kong running prefix and allow for
```

Comment:
"to prepare the Kong running prefix" sounds awkward, but maybe this is correct terminology?